src/macros.rs

```diff
@@ -371,3 +371,9 @@ macro_rules! FN {
         pub type $func = Option<unsafe extern "C" fn($($p: $t,)*) -> $ret>;
     );
 }
+macro_rules! DEFINE_CA {
```

Comment:
This macro is named `TYPEDEF_CA` in the `.idl` file.

Reply:
Also, since it is only used in one module, it should probably be defined in that module too.